04 - Guides, Workflows, & Courses/Guides/🗂️ Guides.md

```diff
@@ -36,7 +36,8 @@ tags:
 
 ## Contributing
 
-1. Create a note under [[🗂️ 04 - Guides, Workflows, & Courses]] or one of it's subfolders
+1. Create a note under one of the subfolders of `04 - Guides, Workflows, & Courses`
 2. Apply the [[T - New Note]] template
-3. Write to your heart's content!
-4. [[How to add content through GitHub|Submit your changes to GitHub]]
\ No newline at end of file
+3. Link the new guide in one overview notes at  [[🗂️ 04 - Guides, Workflows, & Courses]]
```

Comment:
This won't work. Links should go to a manual MOC.

Reply:
fixed